src/main/java/com/barterbay/app/domain/ConfirmationToken.java

```diff
@@ -0,0 +1,64 @@
+package com.barterbay.app.domain;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToOne;
+import javax.persistence.SequenceGenerator;
+import javax.persistence.Table;
+import java.io.Serializable;
+import java.time.LocalDateTime;
+
+@Getter
+@Setter
+@NoArgsConstructor
```

Comment:
IDEA prompts I should have non-arg constructor, so i need it

Reply:
You`ve added this annotation because IDEA just said that? do you use this object somewhere without a contactor? show me the usage of this object WITHOUT constractor